yjit.c

```diff
@@ -1165,8 +1165,8 @@ rb_yjit_invalidate_all_method_lookup_assumptions(void)
 }
 
 // Number of object shapes, which might be useful for investigating YJIT exit reasons.
-static VALUE
-object_shape_count(rb_execution_context_t *ec, VALUE self)
+VALUE
+object_shape_count(void)
```

Comment:
Does this seem like the right change?  We used to use it as a `Primitive`, now we are just calling it from rust.